.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+### [READ] Guidelines
+
+When filing a feature request please make sure the issue title starts with "FR:".
```

Comment:
By the way you can tell GitHub to auto-label issues with a certain template.  So you could give this one `type: feature request` automatically.

See:
https://raw.githubusercontent.com/google/ExoPlayer/release-v2/.github/ISSUE_TEMPLATE/feature_request.md